tests/api/test_workflow.py

```diff
@@ -0,0 +1,11 @@
+import unittest, sys, datetime, copy
+
+sys.path.append("./src")
+
+from api import Workflow
+
+
+class TestWorkflow(unittest.TestCase):
+    def test_run_workflow(self):
+        workflow = Workflow(workflow="./demo/api_demo/demo_workflow.json")
+        workflow.run_workflow(verbose=True)
```

Comment:
This seems to be an integration test. Do we want unit tests as well?

Reply:
Yes you are absolutely right. Unit tests are to be added for workflow API functionalities